index.html

```diff
@@ -1198,51 +1198,33 @@ <h3>Contexts</h3>
         <h3>Identifiers</h3>
 
         <p>
-When expressing statements about a specific thing, such as a person, product,
-or organization, it is often useful to use some kind of identifier so that
-others can express statements about the same thing. This specification defines
-the optional <code>id</code> <a>property</a> for such identifiers. The
-<code>id</code> <a>property</a> is intended to unambiguously refer to an object,
-such as a person, product, or organization. Using the <code>id</code>
-<a>property</a> allows for the expression of statements about specific things in
-the <a>verifiable credential</a>.
+사람, 제품, 또는 조직과 같은 특정한 것에 대해 표현할때, 다른 사람들이 동일한 것을 표현할 수 있도록 식별자를 사용하는 것이 유용하다. 이 규격에서는 이러한 식별자들에 대한 선택적 <code>id</code> 속성을 정의한다. <code>id</code> <a>속성</a>은 사람, 제품 또는 조직과 같은 객체를 명확하게 참조하기 위한 것이다. <code>id</code> <a>속성</a>을 사용하면 <a>검증가능한 크리덴셜</a>을 통해 특정한 사물을 표현할 수 있게 된다.
         </p>
 
         <p>
-<em>If</em> the <code>id</code> <a>property</a> is present:
+<em>만약</em> the <code>id</code> <a>속성</a>이 존재한다면:
         </p>
 
         <ul>
           <li>
-The <code>id</code> <a>property</a> MUST express an identifier that others are
-expected to use when expressing statements about a specific thing identified
-by that identifier.
+The <code>id</code> <a>속성</a>을 표현할 때 반드시 다른 사용자들이 특정 사물을 표현할 때 사용할 것으로 예상되는 식별자로 표현해야 한다.
```

Comment:
속성값에 The id 라고 들어가는게 아니니까 빼는것이 맞을것 같습니다

```suggestion
<code>id</code> <a>속성</a>을 표현할 때 반드시 다른 사용자들이 특정 사물을 표현할 때 사용할 것으로 예상되는 식별자로 표현해야 한다.
```